examples/CampaignManagement/CreateCampaignExperiment.cs

```diff
@@ -0,0 +1,165 @@
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V1.Errors;
+using Google.Ads.GoogleAds.V1.Common;
+using Google.Ads.GoogleAds.V1.Resources;
+using Google.Ads.GoogleAds.V1.Services;
+
+using System;
+using System.Linq;
+using static Google.Ads.GoogleAds.V1.Enums.CampaignExperimentTrafficSplitTypeEnum.Types;
+using Google.Protobuf.WellKnownTypes;
+using Google.LongRunning;
+
+namespace Google.Ads.GoogleAds.Examples.V1
+{
+    /// <summary>
+    /// This code example adds a campaign draft for a campaign. Make sure you specify a
+    /// campaign that has a budget with explicitly_shared set to false.
+    /// </summary>
+    public class CreateCampaignExperiment : ExampleBase
+    {
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            CreateCampaignExperiment codeExample = new CreateCampaignExperiment();
+            Console.WriteLine(codeExample.Description);
+
+            // The Google Ads customer ID for which the call is made.
+            long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+
+            // ID of the base campaign.
+            string campaignDraftResourceName = "INSERT_CAMPAIGN_DRAFT_RESOURCE_ID_HERE";
+
+            codeExample.Run(new GoogleAdsClient(), customerId, campaignDraftResourceName);
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description
+        {
+            get
+            {
+                return "This code example adds a campaign draft for a campaign. Make sure you " +
+                    "specify a campaign that has a budget with explicitly_shared set to false.";
+            }
+        }
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="customerId">The Google Ads customer ID for which the call is made.</param>
+        /// <param name="campaignDraftResourceName">ID of the base campaign</param>
+        public void Run(GoogleAdsClient client, long customerId, string campaignDraftResourceName)
+        {
+            try
+            {
+                LongRunning.Operation<Empty, CreateCampaignExperimentMetadata> operation =
+                    CreateExperiment(client, customerId, campaignDraftResourceName);
+
+                // Use the credentials from the current client.
+                //operation = operation.UseClient(client);
+
+                Console.WriteLine("Waiting until operation completes...");
+                operation.PollUntilCompleted();
+
```

Comment:
Is this and the commenting out below intentional?

Reply:
No, it isn't. Fixed.